### PR TITLE
Adding Manav Kapoor's Profile as a Community Member

### LINKED ISSUE #2404: [Community] Member Profile: Manav Kapoor
**Current Behavior**

@manav1403  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @manav1403 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @manav1403 
- LinkedIn: https://www.linkedin.com/in/manav-kapoor-14a14018a
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Deepansharora27**
**Description**

This PR fixes #2404 

**Notes for Reviewers**

- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Deepansharora27** [comment]
Also, tagging @leecalcote @warunicorn19  for the update

**Comment by warunicorn19** [comment]
@Deepansharora27 Seems like you forgot to sign off😅

**Comment by Deepansharora27** [comment]
@Yeah Sorry for that @warunicorn19 
Solved it 
Will keep that in mind for future PR's

**Comment by Deepansharora27** [comment]
The Preview Build was failing 
I guess the reason for it was that I made a typo while specifying the Path of the Image 
Have Fixed it Now !!

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@Deepansharora27 the build is failing because of the "" used in the bio, could you please remove those.

**Comment by leecalcote** [comment]
@manav1403 🎈 !

**Comment by manav1403** [comment]
> @manav1403 🎈 !

🎉🎉😁

File: src/collections/members/manav-kapoor/index.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Manav Kapoor
+position: Contributor and Community Member
+image_path: ./Manav-kapoor.jpeg
+github: manav1403
+linkedin: manav-kapoor-14a14018a
+status: Active
+location: Jodhpur,Rajasthan,India
+bio: "I love technology, so my work is nothing but my love. #lovefortechnology"
```
Comment (warunicorn19) [inline src/collections/members/manav-kapoor/index.mdx:9]:
```suggestion
bio: I love technology, so my work is nothing but my love. #lovefortechnology
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by Deepansharora27** [comment]
> @Deepansharora27 the build is failing because of the "" used in the bio, could you please remove those.

But, in the Profile Member Template it was mentioned to include the bio in double quotations and if we hop onto @leecalcote profile template as well, his bio is also enclosed within double quotes.

**Comment by warunicorn19** [comment]
Yes, @Deepansharora27 you are right, let me take a look.

**Comment by l5io** [comment]
🚀 Preview for commit 9c677ffc935724c555cb224451bdc90875dc9bfa at: https://61b1a9b7455d75e64e5c1849--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
https://61b1a9b7455d75e64e5c1849--layer5ng.netlify.app/community/members/manav-kapoor

@manav1403^^ waiting for your all-clear.

**Review by manav1403 — APPROVED** [review]

**Comment by manav1403** [comment]
> https://61b1a9b7455d75e64e5c1849--layer5ng.netlify.app/community/members/manav-kapoor
> 
> @manav1403^^ waiting for your all-clear.

Thanks @Deepansharora27 and @warunicorn19 :)

**Review by leecalcote — APPROVED** [review]
Awesome.

thank you, @Deepansharora27

**Pull request merged**